components/LoginButton.js

```diff
@@ -87,7 +87,7 @@ function LoginButton({ isVisible, toggleVisibility }) {
             height: 100%;
           }
           div :global(.profile-button) {
-            max-width: 142px;
+            max-width: 218px;
```

Comment:
@andrewda what is the reason behind this change?

Reply:
Ah good catch, didn't realize I committed that change. My name was cut off to "Andre..." and I was playing with increasing the size a little to allow longer names (this was enough to fit my full name with some room to spare, without any real impact on the UI).

If you'd prefer keep it at 142 though I can definitely revert that change! 👍

Reply:
@andrewda no worries! A little longer is great! Would you mind posting a screenshot?

Reply:
Sure!

## Before

![image](https://user-images.githubusercontent.com/10191084/69904202-d3390280-1358-11ea-807b-c3d27b12072f.png)

## After

![image](https://user-images.githubusercontent.com/10191084/69904181-90772a80-1358-11ea-9893-28017c05efc0.png)